src/utils/sourcemaps.rs

```diff
@@ -611,12 +627,19 @@ impl SourceMapProcessor {
                 strip_prefixes: prefixes,
                 ..Default::default()
             };
-            let sm = match sourcemap::decode_slice(&source.contents)? {
-                sourcemap::DecodedMap::Regular(sm) => sm.rewrite(&options)?,
-                sourcemap::DecodedMap::Index(smi) => smi.flatten_and_rewrite(&options)?,
-            };
             let mut new_source: Vec<u8> = Vec::new();
-            sm.to_writer(&mut new_source)?;
+            match sourcemap::decode_slice(&source.contents)? {
+                sourcemap::DecodedMap::Regular(sm) => {
+                    sm.rewrite(&options)?.to_writer(&mut new_source)?
+                }
+                sourcemap::DecodedMap::Hermes(smh) => {
+                    smh.rewrite(&options)?.to_writer(&mut new_source)?
+                }
+                sourcemap::DecodedMap::Index(smi) => smi
+                    .flatten_and_rewrite(&options)?
+                    .to_writer(&mut new_source)?,
+                _ => panic!("invalid sourcemap type"),
+            };
```

Comment:
You are not writing rewritten sourcemaps to the new source in Regular and Hermes.

Reply:
I am though, or am I missing something?

Reply:
Right, my bad, it got squished into a single line